data_dir/nemesis.yml

```diff
@@ -112,7 +112,7 @@
   - limited = True
 - disrupt_mgmt_restore:
   - disruptive = True
-  - kubernetes = False
```

Comment:
why are touching this one ?

Reply:
I'm guessing someone update it,and didn't run the unittest again....

Reply:
Yes, this diff is in master code.